src/databricks/labs/remorph/snow/dialect_utils.py

```diff
@@ -25,26 +34,34 @@ def check_for_unsupported_lca(
     erroneous_aliases = set()
     for expr in parsed_expr:
         if expr:
-            for select in expr.find_all(exp.Select):
+            for select in expr.find_all(exp.Select, bfs=False):
                 erroneous_aliases.update(_find_invalid_lca_in_select(select))
     if erroneous_aliases:
         err_message = (
             f"Unsupported operation found in file {filename}. "
             f"Lateral column aliases `{', '.join(erroneous_aliases)}` found in "
```

Comment:
since we are specifically checking for where and window can the error message be more specific?

Reply:
Done